include/deal.II/base/index_set.h

```diff
@@ -256,6 +256,14 @@ class IndexSet
   size_type index_within_set (const size_type global_index) const;
 
   /**
+   * Same as above, but returns <code>true</code> if the global index is
+   * within the set and <code>false</code> otherwise. In the former case
+   * the how-manyth element of this set is set in @p local index.
+   */
+  bool index_within_set (const size_type global_index,
+                         size_type &local_index) const;
```

Comment:
I don't like the name of this function. If it returns a bool, it ought to be named something different than `index_` because that would suggest returning an integer. How about `is_index_within_set`?